qiskit/schemas/qobj_schema.json

```diff
@@ -152,6 +153,24 @@
                     ],
                     "title": "Frame change pulse"
                 },
+                {
+                    "properties": {
+                        "name": {
+                            "enum": [
+                                "parametric_pulse"
+                            ]
+                        },
+                        "pulse_shape": {"type": "string"},
+                        "params": {"type": "array"}
```

Comment:
actually a dict, how do I write that here? the keys are strings but the values aren't all the same type, and the keys aren't set either